cpp/ps0000/ps0000solution.cpp

```diff
@@ -1,161 +1,9 @@
-﻿
-#include <iostream>
-#include "hadhod.cpp"
-#include "easylearn.cpp"
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`